engine/install/debian.md

```diff
@@ -31,13 +31,19 @@ architectures.
 
 ### Uninstall old versions
 
-Older versions of Docker went by the names of `docker`, `docker.io`, or
-`docker-engine`, you might also have installations of `containerd` or `runc`.
-Uninstall any such older versions before attempting to install
-a new version:
+Older versions of Docker were called `docker`, `docker.io`, and
```

Comment:
`docker.io` is not an older version, but (for debian and ubuntu) is the name of the package as shipped by the distro maintainers.

I think by this time, we can probably stop mentioning the `docker` and `docker-engine` packages; those were released many years ago, and may not even have been released for any of the currently supported versions of debian and ubuntu.

So, probably the "might conflict" ones are;

- `docker.io` (distro-packaged version of docker engine)
- `containerd` and `runc` (which we bundle together in our `containerd.io` package)

Let me check for others; looking at the list of packages;

```bash
apt-cache search --names-only 'docker'
docker - transitional package
docker-clean - simple Shell script to clean up the Docker Daemon
docker-compose - define and run multi-container Docker applications with YAML
docker-doc - Linux container runtime -- documentation
docker-registry - Docker toolset to pack, ship, store, and deliver content
docker.io - Linux container runtime
docker2aci - CLI tool to convert Docker images to ACIs
elpa-dockerfile-mode - Major mode for editing Docker's Dockerfiles
golang-docker-credential-helpers - native stores to safeguard Docker credentials
golang-docker-dev - Transitional package for golang-github-docker-docker-dev
golang-github-appc-docker2aci-dev - library to convert Docker images to ACIs
golang-github-docker-distribution-dev - Docker toolset to pack, ship, store, and deliver content (source)
golang-github-docker-docker-credential-helpers-dev - native stores to safeguard Docker credentials - library
golang-github-docker-docker-dev - Externally reusable Go packages included with Docker
golang-github-docker-go-connections-dev - Golang utility package to work with network connections
golang-github-docker-go-dev - Go packages with small patches autogenerated (used for canonical/json)
golang-github-docker-go-events-dev - Composable event distribution for Go
golang-github-docker-go-metrics-dev - Package for metrics collection in Docker projects
golang-github-docker-go-units-dev - parse and print size and time units in human-readable format
golang-github-docker-leadership-dev - distributed leader election using docker/libkv
golang-github-docker-libkv-dev - Key/Value store abstraction library
golang-github-docker-libtrust-dev - Primitives for identity and authorization
golang-github-docker-notary-dev - library for running and interacting with trusted collections
golang-github-docker-spdystream-dev - multiplexed stream library using spdy
golang-github-fsouza-go-dockerclient-dev - Docker client library in Go
golang-github-samalba-dockerclient-dev - Docker client library in Go
kdocker - lets you dock any application into the system tray
libnss-docker - nss module for finding Docker containers
ovn-docker - OVN Docker drivers
podman-docker - engine to run OCI-based containers in Pods - wrapper for docker
python3-docker - Python 3 wrapper to access docker.io's control socket
python3-dockerpty - Pseudo-tty handler for docker Python client (Python 3.x)
python3-dockerpycreds - Python3 bindings for the docker credentials store API
ruby-docker-api - Ruby gem to interact with docker.io remote API
ruby-kitchen-docker - Docker Driver for Test Kitchen
vim-syntax-docker - Docker container engine - Vim highlighting syntax files
wmdocker - System tray for KDE3/GNOME2 docklet applications
```



Ah; couple more:

```
docker-compose - define and run multi-container Docker applications with YAML
docker-doc - Linux container runtime -- documentation
docker.io - Linux container runtime
podman-docker - engine to run OCI-based containers in Pods - wrapper for docker
```